tl/appendices/3-0-migration-guide.rst

```diff
@@ -944,156 +946,177 @@ Helper has had the following methods removed:
-  Use ``{{page}}`` style placeholders instead.
-- ``url()`` has been renamed to ``generateUrl()`` to avoid method declaration clashes with ``Helper::url()``.
-
-By default all links and inactive texts are wrapped in ``<li>`` elements. This
-helps make CSS easier to write, and improves compatibility with popular CSS
-frameworks.
-
-Instead of the various options in each method, you should use the templates
-feature. See the :ref:`paginator-templates` documentation for
-information on how to use templates.
+- Ang ``link()`` ay tinanggal. Ito ay hindi na panloob na ginagamit ng helper.
+  Ito ay may mababang paggamit sa user land code, at hindi na kasya sa mga 
+  layunin ng helper.
+- Ang ``next()`` ay wala nang 'class', o 'tag' na mga opsyon. Wala na itong naka-disable
+  na mga argumento. Sa halip ay ginamit ang mga template.
+- Ang ``prev()`` ay wala nang 'class', o 'tag' na mga opsyon. Wala na itong naka-disable
+  na mga argumento. Sa halip ay ginamit ang mga template.
+- Ang ``first()`` ay wala nang 'after', 'ellipsis', 'separator', 'class', o 'tag' na mga opsyon.
+- Ang ``last()`` ay wala nang 'after', 'ellipsis', 'separator', 'class', o 'tag' na mga opsyon.
+- Ang ``numbers()`` ay wala nang 'separator', 'tag', 'currentTag', 'currentClass',
+  'class', 'tag', 'ellipsis' na mga opsyon. Ang mga opsyon na ito ay pinadali na ngayon gamit 
+  ang mga template. Ito rin ay nangangailangan ng ``$options`` na parameter na maging 
+  isang array na ngayon.
+- Ang ``%page%`` na estilo na mga placeholder ay tinanggal mula sa 
+  :php:meth:`Cake\\View\\Helper\\PaginatorHelper::counter()`.
+  Sa halip ay gamitin ang ``{{page}}`` na estilo na mga placeholder.
+- Ang ``url()`` ay napalitan ang pangaln sa ``generateUrl()`` upang maiwasan ang banggaan sa deklarasyon ng paraan
+  gamit ang ``Helper::url()``.
+
+Bilang default ang lahat ng mga link at hindi aktibong mga teksto ay nakabalot sa ``<li>`` na
+nga elemento. Tinutulungan nitong gawing mas madali ang pagsulat ng CSS, at papabutihin ang
+pagkakatugma sa popular na mga balangkas ng CSS.
+
+Sa halip ng iba't ibang mga opsyon sa bawat paraan, dapat mong gamitin ang mga template
+na tampok. Tingnan ang :ref:`paginator-templates` dokumentasyon para sa impormasyon
+kung paano gamitin ang mga template.
 
 TimeHelper
 ----------
 
-- ``TimeHelper::__set()``, ``TimeHelper::__get()``, and  ``TimeHelper::__isset()`` were
-  removed. These were magic methods for deprecated attributes.
-- ``TimeHelper::serverOffset()`` has been removed.  It promoted incorrect time math practices.
-- ``TimeHelper::niceShort()`` has been removed.
+- Ang ``TimeHelper::__set()``, ``TimeHelper::__get()``, at  ``TimeHelper::__isset()`` ay
+  tinanggal. Ito ay ang madyik na mga paraan para sa hindi na nagagamit na mga katangian.
+- Ang ``TimeHelper::serverOffset()`` ay tinanggal. Ito ay nagtataguyod ng hindi wastong
+  time match na mga gawi.
```

Comment:
time match to time na tugma

Reply:
changing it to tugmang oras instead. thanks